src/QMCWaveFunctions/BsplineFactory/SplineR2R.h

```diff
@@ -79,8 +80,15 @@ class SplineR2R : public BsplineSet
     KeyWord    = "SplineR2R";
   }
 
+  // Sets the number of splines
+  // NB: This may be <= number of splined orbs from SplineInst
+  void setOrbitalSetSize(int norbs) override { OrbitalSetSize = norbs; }
```

Comment:
What?  I don't understand why the the BSplineSet member function needs to be duplicated here.

Reply:
Oops. I didn't notice that it was already defined elsewhere. Removed the superfluous definition in SplineR2R.cpp